plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/ExtendStore.php

```diff
@@ -35,6 +35,18 @@ public function get_content() {
 		return '';
 	}
 
+	/**
+	 * Additional info.
+	 *
+	 * @return string
+	 */
+	public function get_additional_info() {
```

Comment:
TIL this comes from [`Task`](https://github.com/woocommerce/woocommerce/blob/be2d69c130db75b76d186f369b70e32115f98283/plugins/woocommerce/src/Admin/Features/OnboardingTasks/Task.php#L511) 👍